website/docs/r/upgrade_run.html.markdown

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Beta"
+layout: "nsxt"
+page_title: "NSXT: nsxt_upgrade_run"
+description: A resource to configure and execute upgrade of NSXT cluster.
+---
+
+# nsxt_upgrade_run
+
+This resource provides a method to configure and execute upgrade of NSXT cluster.
+
+It will first configure upgrade unit groups and upgrade plan settings for `EDGE`
+and `HOST` components. Then it will execute the upgrade for each component. If 
+there are either `EDGE` or `HOST` groups disabled from the upgrade, the corresponding 
+component will be paused after enabled groups get upgraded. At the same time, `MP` upgrade
+won't be processed, because that requires `EDGE` and `HOST` to be fully upgraded. Refer to
+the exposed attribute for current upgrade state details.
+
+If upgrade post-checks are configured to be run, it will trigger the upgrade post-check.
+Please use data source `nsxt_upgrade_postcheck` to retrieve results of upgrade post-checks.
+
+## Example Usage
+
+```hcl
+resource "nsxt_upgrade_run" "run1" {
+  upgrade_prepare_id = nsxt_upgrade_prepare.test.id
+
+  edge_group {
+    id      = data.nsxt_edge_upgrade_group.eg1.id
+    enabled = false
+  }
+
+  edge_group {
+    id      = data.nsxt_edge_upgrade_group.eg2.id
+    enabled = true
+  }
+
+  host_group {
+    id       = data.nsxt_edge_upgrade_group.eg2.id
```

Comment:
this should be data.nsxt_host_upgrade_group

Reply:
Done.